packages/kit/src/components/UpdateReminder/hooks.ts

```
@@ -36,39 +36,98 @@ export const useAppChangeLog = (version?: string) => {
-            ...result,
-          }),
-          timerUtils.wait(4500),
-        ]);
-        await backgroundApiProxy.serviceAppUpdate.readyToInstall();
+        await backgroundApiProxy.serviceAppUpdate.updateDownloadedEvent({
+          ...params,
+          ...result,
+        });
+        await downloadASC();
       } catch (e) {
+        await backgroundApiProxy.serviceAppUpdate.downloadPackageFailed(e as Error);
```

ESLint

[error] Replace `e·as·Error` with `⏎··········e·as·Error,⏎········`

(prettier/prettier)